### DISCUSSION TITLE
Issues running Bun instead of Node?

### DISCUSSION THREAD

**Discussion by e253** [testcontainers/testcontainers-node#655] [General]
I've been trying to use testcontainers with an [elysiajs](elysiajs.com) application. I'm having problems detecting the docker runtime. I haven't had issues with python so I think it might be the bun substitution causing the problem. 

Has anyone been able to get testcontainers to work with Bun?

I am using wsl on a windows system.

**Comment by kiview** [comment]
> I haven't had issues with python

tc-node and tc-python don't share code or implementation details. The best way to check whether this issue is related to Bun would be to try out tc-node in your environment using Node.
